app/Http/Controllers/UsersController.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace App\Http\Controllers;
+use App\Http\Requests;
+
+use App\User;
+
+class UsersController extends Controller
+{
+
+    public function index()
+    {
+        $users = User::all();
+        return view('users.index', compact('users'));
+    }
+
+    public function show($id)
+    {
+        $user = User::findOrfail($id);
```

Comment:
@mansoursaid001  Also do you know any other alternative for the try&catch block? I searching but I thought I should also ask you.